edx_when/models.py

```diff
@@ -54,6 +54,11 @@ def actual_date(self, schedule=None, end_datetime=None):
                         self
                     )
                 )
+
+            # If the user enrolled after the course ended, we don't want to return any dates.
+            if end_datetime and schedule.start_date > end_datetime:
```

Comment:
I think this makes sense. I forget whether we ceded some control over when relative dates are returned to the LMS/caller of edx-when? If we did, that could be useful to use in the future if we want to manage how dates are returned when the schedule is in the grey area late in the course. But for now, I think this is a good step.